templates/extender/deployment.yaml

```diff
@@ -0,0 +1,75 @@
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  labels:
+    component: scheduler
+    tier: control-plane
+  name: {{ include "extender.name" . }}
+  namespace: kube-system
+spec:
+  selector:
+    matchLabels:
+      app: {{ include "extender.name" . }}
+      component: scheduler
+      tier: control-plane
+  replicas: {{ .Values.scheduler.replicas }}
+  template:
+    metadata:
+      labels:
+        app: {{ include "extender.name" . }}
+        component: scheduler
+        tier: control-plane
+        version: second
+    spec:
+      serviceAccountName: {{ include "extender.name" . }}
+      containers:
+        - name: kube-{{ include "scheduler.name" . }}
+          image: public.ecr.aws/eks-distro/kubernetes/kube-scheduler:v{{(semver .Capabilities.KubeVersion.GitVersion).Major}}.{{(semver .Capabilities.KubeVersion.GitVersion).Minor}}.0-eks-{{(semver .Capabilities.KubeVersion.GitVersion).Major}}-{{(semver .Capabilities.KubeVersion.GitVersion).Minor}}-latest
```

Comment:
would suggest making this in tpl

Reply:
will do it first in dev and than take it to prod